### ISSUE TITLE
No local site generation possible (like a yarn antora-local command)

### ISSUE THREAD

**Issue by mmattel** [owncloud/docs#684]
## What Needs to be Documented?
Refrencing https://github.com/owncloud/docs/blob/master/docs/build-the-docs.md#using-antora-from-the-command-line you see, it is described that running `yarn antora` generates a **local** version of the documentation. In reality it is creating the version from the **git** repository.
We had in the past a make command which differenciated between git and local, see issue und PR´s https://github.com/owncloud/docs/issues/486

## Changes needed
Implement a possibility to create a local version of the documentation like with a `yarn antora-local` command.

## Why Should This Change Be Made? (Optional)

Else you can never fully test local changes.

## What Type Of Content Change Is This? (Optional)
- [ ] New Content Addition
- [ ] Old Content Deprecation
- [ ] Existing Content Simplification
- [x] Bug Fix to Existing Content

## Which Manual Does This Relate To? (Optional)
- [ ] Admin Manual
- [ ] Developer Manual
- [ ] User Manual
- [ ] Android
- [ ] iOS
- [ ] Branded Clients
- [ ] Desktop Client
- [x] Other

@settermjd @tboerger

**Comment by tboerger** [comment]
You don't need any local config because the site config uses the current repo. Just append your local url to yarn antora as documented.

**Comment by mmattel** [comment]
ok, let me tell you the problem in another way.
Any local changes made can not be build to a reviewable site as the current way takes always the source from github.

**Comment by settermjd** [comment]
@tboerger, @mmattel is correct. The local playbook, `site.local.yml`, needs to be returned, as `site.yml` doesn't activate [Author Mode](https://docs.antora.org/antora/2.0/playbook/author-mode/); it only draws content to compile from the remote repositories.

**Comment by tboerger** [comment]
I was told that https://github.com/owncloud/docs/blob/master/site.yml#L7 is enough. If it requires branch: HEAD just add it there, there is absolutely no need for another site config.

**Comment by settermjd** [comment]
Is it? That line seemed like a mistake, and it's not covered in the documentation.

**Comment by tboerger** [comment]
It's not a mistake

**Comment by settermjd** [comment]
As I said, because there's nothing in the documentation, it seems so.

**Comment by mmattel** [comment]
@tboerger I do not open an issue if things are working. I open an issue if things do not work which is the case here. So please be open to someone elses finding even if you have other experiences.
I can tell you, that I have the total opposite experience as you have - so let´s identify the cause and fix the issue, thanks.

**Comment by tboerger** [comment]
I have shown exactly above that it works like expected, that's why I have closed the issue. I will unsubscribe from this repo and you can take care on your own.

**Comment by mmattel** [comment]
Just because we have different experiences?
Your findings and support is always helpful !
It is about cooperation - this is what you teached me.

**Comment by tboerger** [comment]
I have clearly shown that it works. You just said it doesn't work without proving anything. So why should I care about that anymore?

**Comment by mmattel** [comment]
Ubuntu 18.04
```
git diff
diff --git a/modules/ROOT/pages/release_notes.adoc b/modules/ROOT/pages/release_notes.adoc
index 31d7e7be..83337ca1 100644
--- a/modules/ROOT/pages/release_notes.adoc
+++ b/modules/ROOT/pages/release_notes.adoc
@@ -5,3 +5,4 @@
 ** Page Edit
 * 2018 December
 ** Initial Release based on Antora
+Will it work?
```
```
yarn antora --url http://localhost:5000yarn run v1.13.0
$ antora --stacktrace generate --cache-dir cache --redirect-facility disabled --generator ./generator/generate-site.js --clean site.yml --url http://localhost:5000
```
```
grep -R "Will it work" public/
<returns no result>
```
Also opening the release_notes.html file and searching for "Will" does not return anything.

So we do have two different results for the same procedure.
This means we have to care.

**Comment by tboerger** [comment]
Are you really running the latest defined antora version?

**Comment by mmattel** [comment]
I setup a fresh new Ubuntu 18.04 instance about 10 days ago including my dev environment.
Cloning docs and running make setup, everything looks quite normal and ok.

**Comment by tboerger** [comment]
> The setup as configured can´t automatically differenciate between external and author site building - how should it, based on which criteria?

Of course it can, it's done like that by Drone and also by others, not only me. The site.yml is referencing `.` which is the current working directory and additionally with another step the fixes branches 10.0 and 10.1 within another step.

**Comment by settermjd** [comment]
I just created #720 to resolve this.

**Comment by mmattel** [comment]
Closing, fixed by PR #720
